<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>modifyUser</class>
    <widget class="QDialog" name="modifyUser">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1118</width>
                <height>417</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <widget class="QLabel" name="infoLabel">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>80</y>
                    <width>901</width>
                    <height>81</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Please fill in these fields and click on the &quot;Create an account&quot; button to let L-Book
                    configure the new account.
                </string>
            </property>
            <property name="alignment">
                <set>Qt::AlignJustify|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
                <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
                <set>Qt::NoTextInteraction</set>
            </property>
        </widget>
        <widget class="QLabel" name="logoLabel">
            <property name="geometry">
                <rect>
                    <x>-10</x>
                    <y>10</y>
                    <width>1131</width>
                    <height>101</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>A Charming Font Expanded</family>
                    <pointsize>36</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                </font>
            </property>
            <property name="text">
                <string>L-Book</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
            <property name="textInteractionFlags">
                <set>Qt::NoTextInteraction</set>
            </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_1">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>160</y>
                    <width>1071</width>
                    <height>161</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                    <widget class="QGroupBox" name="groupBox">
                        <property name="title">
                            <string>Account:</string>
                        </property>
                        <widget class="QWidget" name="layoutWidget_2">
                            <property name="geometry">
                                <rect>
                                    <x>20</x>
                                    <y>30</y>
                                    <width>215</width>
                                    <height>111</height>
                                </rect>
                            </property>
                            <layout class="QVBoxLayout" name="verticalLayout_11">
                                <item>
                                    <layout class="QHBoxLayout" name="horizontalLayout">
                                        <item>
                                            <layout class="QVBoxLayout" name="verticalLayout_10">
                                                <item>
                                                    <widget class="QLabel" name="label">
                                                        <property name="text">
                                                            <string>Username:</string>
                                                        </property>
                                                        <property name="textInteractionFlags">
                                                            <set>Qt::NoTextInteraction</set>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QLabel" name="label_2">
                                                        <property name="text">
                                                            <string>Password:</string>
                                                        </property>
                                                        <property name="textInteractionFlags">
                                                            <set>Qt::NoTextInteraction</set>
                                                        </property>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </item>
                                        <item>
                                            <layout class="QVBoxLayout" name="verticalLayout">
                                                <item>
                                                    <widget class="QLineEdit" name="usernameLine"/>
                                                </item>
                                                <item>
                                                    <widget class="QLineEdit" name="passwordLine">
                                                        <property name="echoMode">
                                                            <enum>QLineEdit::Password</enum>
                                                        </property>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </item>
                                    </layout>
                                </item>
                            </layout>
                        </widget>
                    </widget>
                </item>
                <item>
                    <widget class="QGroupBox" name="groupBox_2">
                        <property name="title">
                            <string>Information:</string>
                        </property>
                        <widget class="QWidget" name="layoutWidget_3">
                            <property name="geometry">
                                <rect>
                                    <x>20</x>
                                    <y>30</y>
                                    <width>221</width>
                                    <height>113</height>
                                </rect>
                            </property>
                            <layout class="QHBoxLayout" name="horizontalLayout_3">
                                <item>
                                    <layout class="QVBoxLayout" name="verticalLayout_5">
                                        <item>
                                            <widget class="QLabel" name="label_3">
                                                <property name="text">
                                                    <string>Name:</string>
                                                </property>
                                                <property name="textInteractionFlags">
                                                    <set>Qt::NoTextInteraction</set>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QLabel" name="label_4">
                                                <property name="text">
                                                    <string>Surname:</string>
                                                </property>
                                                <property name="textInteractionFlags">
                                                    <set>Qt::NoTextInteraction</set>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QLabel" name="label_5">
                                                <property name="text">
                                                    <string>Birthdate:</string>
                                                </property>
                                                <property name="textInteractionFlags">
                                                    <set>Qt::NoTextInteraction</set>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QLabel" name="label_6">
                                                <property name="text">
                                                    <string>Class:</string>
                                                </property>
                                                <property name="textInteractionFlags">
                                                    <set>Qt::NoTextInteraction</set>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                                <item>
                                    <layout class="QVBoxLayout" name="verticalLayout_2">
                                        <item>
                                            <widget class="QLineEdit" name="nameLine"/>
                                        </item>
                                        <item>
                                            <widget class="QLineEdit" name="surnameLine"/>
                                        </item>
                                        <item>
                                            <widget class="QDateEdit" name="dateEdit"/>
                                        </item>
                                        <item>
                                            <widget class="QLineEdit" name="classLine"/>
                                        </item>
                                    </layout>
                                </item>
                            </layout>
                        </widget>
                    </widget>
                </item>
                <item>
                    <widget class="QGroupBox" name="groupBox_4">
                        <property name="title">
                            <string>Contacts:</string>
                        </property>
                        <widget class="QWidget" name="layoutWidget_4">
                            <property name="geometry">
                                <rect>
                                    <x>20</x>
                                    <y>30</y>
                                    <width>222</width>
                                    <height>111</height>
                                </rect>
                            </property>
                            <layout class="QVBoxLayout" name="verticalLayout_7">
                                <item>
                                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                                        <item>
                                            <layout class="QVBoxLayout" name="verticalLayout_6">
                                                <item>
                                                    <widget class="QLabel" name="label_9">
                                                        <property name="text">
                                                            <string>Cellnumber:</string>
                                                        </property>
                                                        <property name="textInteractionFlags">
                                                            <set>Qt::NoTextInteraction</set>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QLabel" name="label_10">
                                                        <property name="text">
                                                            <string>Email:</string>
                                                        </property>
                                                        <property name="wordWrap">
                                                            <bool>false</bool>
                                                        </property>
                                                        <property name="textInteractionFlags">
                                                            <set>Qt::NoTextInteraction</set>
                                                        </property>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </item>
                                        <item>
                                            <layout class="QVBoxLayout" name="verticalLayout_4">
                                                <item>
                                                    <widget class="QLineEdit" name="cellLine"/>
                                                </item>
                                                <item>
                                                    <widget class="QLineEdit" name="mailLine"/>
                                                </item>
                                            </layout>
                                        </item>
                                    </layout>
                                </item>
                            </layout>
                        </widget>
                    </widget>
                </item>
                <item>
                    <widget class="QGroupBox" name="groupBox_3">
                        <property name="title">
                            <string>Permission:</string>
                        </property>
                        <widget class="QWidget" name="layoutWidget_5">
                            <property name="geometry">
                                <rect>
                                    <x>30</x>
                                    <y>30</y>
                                    <width>201</width>
                                    <height>111</height>
                                </rect>
                            </property>
                            <layout class="QVBoxLayout" name="verticalLayout_9">
                                <item>
                                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                                        <item>
                                            <layout class="QVBoxLayout" name="verticalLayout_8">
                                                <item>
                                                    <widget class="QLabel" name="label_7">
                                                        <property name="text">
                                                            <string>Type:</string>
                                                        </property>
                                                        <property name="textInteractionFlags">
                                                            <set>Qt::NoTextInteraction</set>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QLabel" name="label_8">
                                                        <property name="text">
                                                            <string>Loan:</string>
                                                        </property>
                                                        <property name="textInteractionFlags">
                                                            <set>Qt::NoTextInteraction</set>
                                                        </property>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </item>
                                        <item>
                                            <layout class="QVBoxLayout" name="verticalLayout_3">
                                                <item>
                                                    <widget class="QComboBox" name="comboBox">
                                                        <property name="font">
                                                            <font>
                                                                <pointsize>8</pointsize>
                                                            </font>
                                                        </property>
                                                        <item>
                                                            <property name="text">
                                                                <string>Student</string>
                                                            </property>
                                                        </item>
                                                        <item>
                                                            <property name="text">
                                                                <string>Professor</string>
                                                            </property>
                                                        </item>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QLineEdit" name="loanLine">
                                                        <property name="maximumSize">
                                                            <size>
                                                                <width>16777215</width>
                                                                <height>16777215</height>
                                                            </size>
                                                        </property>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </item>
                                    </layout>
                                </item>
                            </layout>
                        </widget>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_5">
            <property name="geometry">
                <rect>
                    <x>61</x>
                    <y>341</y>
                    <width>500</width>
                    <height>59</height>
                </rect>
            </property>
            <property name="minimumSize">
                <size>
                    <width>500</width>
                    <height>0</height>
                </size>
            </property>
            <property name="title">
                <string>Note:</string>
            </property>
            <widget class="QLabel" name="noteLabel">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>20</y>
                        <width>471</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Name, Surname and Birthdare are required. if you put username you have also to put the
                        password. Username cannot be changed after registration.
                    </string>
                </property>
                <property name="alignment">
                    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
                </property>
                <property name="wordWrap">
                    <bool>true</bool>
                </property>
                <property name="textInteractionFlags">
                    <set>Qt::NoTextInteraction</set>
                </property>
            </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget_6">
            <property name="geometry">
                <rect>
                    <x>805</x>
                    <y>350</y>
                    <width>291</width>
                    <height>32</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                    <widget class="QPushButton" name="nextButton">
                        <property name="maximumSize">
                            <size>
                                <width>250</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Create an account</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="ExitButton">
                        <property name="maximumSize">
                            <size>
                                <width>250</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Exit</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
